### PR TITLE
Add example parsets to docs.

### PR DESCRIPTION
**Pull request by JSKenyon**

### CODE REVIEW

**Comment by JSKenyon** [comment]
ok to test

File: docs/examples.rst [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+
+.. code-block::
+
+    [data]
+    _Help = Visibility data options
+    ms = ms.ms                              # Your measurement set name.
+    column = DATA                           # Column in which the data lives.
+    time-chunk = 100                        # Number of time slots in a chunk. 
+                                            # Implicitly controls memory footprint.
+    freq-chunk = 64                         # Number of frequencies in a chunk.
```
Comment (KshitijT) [inline docs/examples.rst:105]: "Number of frequency channels in a chunk" ?

File: docs/examples.rst [hunk 2]
```diff
@@ -0,0 +1,368 @@
...
```
Comment (KshitijT) [inline docs/examples.rst:182]: "Number of frequency channels in a chunk"; here and elsewhere.

File: docs/examples.rst [hunk 3]
```diff
@@ -0,0 +1,368 @@
...
+    safe = 1.0                              # Fraction of total memory CubiCal may use.
+                                            # Will crash deliberately if exceeded.
+
+    [g]
+    _Help = Options for G-Jones term
+    label = G                               # This term's name.
+                                            # Must match [sol] section's jones option.
+    type = complex-2x2                      # Solve for a full 2x2 complex gain. 
+                                            # This can be restircted using update type.
+    time-int = 0                            # Time interval for solution.
```
Comment (KshitijT) [inline docs/examples.rst:241]: Better to say timeslots?

File: docs/examples.rst [hunk 4]
```diff
@@ -0,0 +1,368 @@
...
+                                            # 0 is the entire chunk axis.
+    freq-int = 1                            # Frequency interval for solution.
```
Comment (KshitijT) [inline docs/examples.rst:243]: Number of channels (by default at least?)

File: docs/examples.rst [hunk 5]
```diff
@@ -0,0 +1,368 @@
...
+                                            # May be ignored if [sol] term-iters is set.
+    _Templated = True                       # Other parameters polulated from defaults.
+                                            # Can be safely ignored.
+
+    [b]
+    _Help = Options for G-Jones term
+    label = b                               # This term's name.
+                                            # Must match [sol] section's jones option.
+    type = complex-2x2                      # Solve for a full 2x2 complex gain. 
+                                            # This can be restircted using update type.
```
Comment (KshitijT) [inline docs/examples.rst]: Typo: "Restricted"

File: docs/examples.rst [hunk 6]
```diff
@@ -0,0 +1,368 @@
...
+    max-iter = 20                           # Maximum number of iterations.
+                                            # May be ignored if [sol] term-iters is set.
+    _Templated = True                       # Other parameters polulated from defaults.
+                                            # Can be safely ignored.
+
+    [de]
+    _Help = Options for G-Jones term
+    label = dE                              # This term's name.
+                                            # Must match [sol] section's jones option.
+    type = complex-2x2                      # Solve for a full 2x2 complex gain.
```
Comment (KshitijT) [inline docs/examples.rst:385]: Can we have an example of the update type which is different from complex-2x2?

**Review by KshitijT — CHANGES_REQUESTED** [review]
Looks really good, thanks !
Just some very minor changes requested, good to go otherwise.

File: docs/examples.rst (hunk 1 above)
Comment (IanHeywood) [reply docs/examples.rst:105]: I've never actually used it in this way, but can't you also specify units here (`Hz`, and and also `s` for time chunks / solution intervals)? Current or suggested help string wouldn't be applicable if so...

Comment (JSKenyon) [reply docs/examples.rst:105]: It does work both ways, but I wanted to strip out as many options/explanations as possible. Will go with the simpler integers for now. These examples don't have to be set in stone, so if users trip up on this we can change/update them.

**Comment by JSKenyon** [comment]
@KshitijT I believe I have addressed your comments. Can you give it a final once over please?

**Comment by JSKenyon** [comment]
retest this please

**Comment by JSKenyon** [comment]
retest this please

**Review by KshitijT — APPROVED** [review]

**Comment by JSKenyon** [comment]
retest this please

**Pull request merged**